src/features/events/store.ts

```diff
@@ -515,32 +515,50 @@ const eventsSlice = createSlice({
 
 function addEventToState(state: EventsStoreSlice, events: ZetkinEvent[]) {
   events.forEach((event) => {
-    const dateStr = event.start_time.slice(0, 10);
-
-    if (!state.eventsByDate[dateStr]) {
-      state.eventsByDate[dateStr] = remoteList();
+    const newDateStr = event.start_time.slice(0, 10);
+    if (!state.eventsByDate[newDateStr]) {
+      state.eventsByDate[newDateStr] = remoteList();
     }
 
     const eventListItem = state.eventList.items.find(
       (item) => item.id == event.id
     );
-    const eventByDateItem = state.eventsByDate[dateStr].items.find(
-      (item) => item.id == event.id
-    );
 
     if (eventListItem) {
       eventListItem.data = { ...eventListItem.data, ...event };
     } else {
       state.eventList.items.push(remoteItem(event.id, { data: event }));
     }
 
-    if (eventByDateItem) {
-      eventByDateItem.data = { ...eventByDateItem.data, ...event };
-      eventByDateItem.mutating = [];
-    } else {
-      state.eventsByDate[dateStr].items.push(
+    let targetedEventDateToUpdate: string | undefined = undefined;
```

Comment:
This name is quite difficult for my eyes to read. Since the other date variable is called `newDateStr`, how about calling this one `oldDateStr` or something? It would make it easier to connect the dots between the two

Reply:
Yes! I've been thinking about to name it `newDateStr` before, but I changed it since it is not very suitable. `oldDateStr` makes sense and sounds better!